<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ПО &quot;Комплекс метрологический автоматизированный &quot;Комета&quot;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройки</string>
    </property>
    <addaction name="settings"/>
    <addaction name="separator"/>
    <addaction name="menu_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Отчеты</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="separator"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Справочники</string>
    </property>
    <addaction name="action_11"/>
    <addaction name="action_12"/>
    <addaction name="action_13"/>
    <addaction name="action_14"/>
    <addaction name="action_15"/>
    <addaction name="separator"/>
    <addaction name="action_17"/>
    <addaction name="action_18"/>
    <addaction name="action_19"/>
    <addaction name="action_9"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="settings">
   <property name="text">
    <string>Общие настройки</string>
   </property>
  </action>
  <action name="menu_exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>прирл</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Эталоны</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>СИ</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>ВО</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>ИО</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Климатические условия</string>
   </property>
  </action>
  <action name="action_7">
   <property name="text">
    <string>Перечень СИ</string>
   </property>
  </action>
  <action name="action_8">
   <property name="text">
    <string>График ТО</string>
   </property>
  </action>
  <action name="action_11">
   <property name="text">
    <string>Реестр УТ СИ</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>Реестр ГЭТ</string>
   </property>
  </action>
  <action name="action_13">
   <property name="text">
    <string>Реестр эталонов ЕВ</string>
   </property>
  </action>
  <action name="action_14">
   <property name="text">
    <string>Реестр УТ СО</string>
   </property>
  </action>
  <action name="action_15">
   <property name="text">
    <string>Перечень СИ в качестве эталонов</string>
   </property>
  </action>
  <action name="action_17">
   <property name="text">
    <string>Структурные подразделения</string>
   </property>
  </action>
  <action name="action_18">
   <property name="text">
    <string>Помещения</string>
   </property>
  </action>
  <action name="action_19">
   <property name="text">
    <string>Сотрудники</string>
   </property>
  </action>
  <action name="action_9">
   <property name="text">
    <string>Информация о предприятии</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>menu_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
